src/main/resources/assets/betterloadingscreen/lang/en_US.lang

```diff
@@ -5,6 +5,7 @@ betterloadingscreen.state.initialization=Initialization
 betterloadingscreen.state.post_initialization=Post Initialization
 betterloadingscreen.state.completed=Completed
 betterloadingscreen.state.reloading_resource_packs=Reloading Resource Packs
+betterloadingscreen.memory_usage=Memory usage: %5d/%5d MB
```

Comment:
Nit: Technically that should be MiB, but I've seen MB used in other Forge packs.

Reply:
I agree for technical cases, but most players probably won't know the difference/care, and MiB is less wide-spread